### PR TITLE
angular 13 upgrade

### PR DESCRIPTION
**Pull request by ArulRozario**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -28,18 +28,6 @@
...
-        },
-        "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
-          "options": {
-            "tsConfig": [
-              "projects/angular-formio/tsconfig.lib.json",
-              "projects/angular-formio/tsconfig.spec.json"
-            ],
-            "exclude": [
-              "**/node_modules/**"
```
Comment (ArulRozario) [inline angular.json:40]: Why was this removed

File: angular.json [hunk 2]
```diff
@@ -28,18 +28,6 @@
             "tsConfig": "projects/angular-formio/tsconfig.spec.json",
             "karmaConfig": "projects/angular-formio/karma.conf.js"
           }
-        },
-        "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
-          "options": {
-            "tsConfig": [
-              "projects/angular-formio/tsconfig.lib.json",
-              "projects/angular-formio/tsconfig.spec.json"
```
Comment (ArulRozario) [inline angular.json:37]: It doesn't seem to be a good one

Comment (ArulRozario) [reply angular.json:37]: I think it needs to re thought

Comment (ArulRozario) [reply angular.json:37]: > I think it needs to re thought

Yeah

**Review by iangabrielsanchez — APPROVED** [review]
